gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -18,19 +18,41 @@
 
 import com.google.cloud.hadoop.gcsio.GoogleCloudStorageReadOptions;
 import com.google.common.base.Preconditions;
+import com.google.common.base.Stopwatch;
 import com.google.common.flogger.GoogleLogger;
+import com.google.common.flogger.LazyArgs;
+import com.google.gson.Gson;
 import java.io.IOException;
 import java.net.URI;
 import java.nio.ByteBuffer;
 import java.nio.channels.ClosedChannelException;
 import java.nio.channels.SeekableByteChannel;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
 import org.apache.hadoop.fs.FSInputStream;
 import org.apache.hadoop.fs.FileSystem;
 
 /** A seekable and positionable FSInputStream that provides read access to a file. */
 class GoogleHadoopFSInputStream extends FSInputStream {
 
   private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
+  static final String READ_METHOD = "gcsFSRead";
```

Comment:
can this be private?

Reply:
marked all of them with @VisibleForTesting